docs/source/developer/xkcd_extension_tutorial.rst

```diff
@@ -52,7 +52,7 @@ Next create a conda environment that includes:
 
 1. the latest release of JupyterLab
 2. `cookiecutter <https://github.com/audreyr/cookiecutter>`__, the tool
-   you'll use to bootstrap your extension project structure
+   you'll use to bootstrap your extension project structure (this is a Python tool, do not try to install with `npm`)
```

Comment:
Thanks! Perhaps we can give users a more positive suggestion, like 
```
(you can install it using `pip install cookiecutter`)
```
?

Reply:
OK sure, I did not at first because I suspected maybe it would lead people away from `conda` ..

will do right away

Reply:
We can suggest conda as well, just didn't want to leave  people with a discouragement from `npm` with no suggestion of what to actually do.